L1Trigger/L1THGCal/interface/backend_emulator/HGCalCluster_SA.h

```diff
@@ -35,11 +35,45 @@ namespace l1thgcfirmware {
                   wroz2_(0),
                   layerbits_(0),
                   sat_tc_(false),
-                  shapeq_(1)
+                  shapeq_(1),
+                  Sigma_E_Quotient_(0),
+                  Sigma_E_Fraction_(0),
+                  Mean_z_Quotient_(0),
+                  Mean_z_Fraction_(0),
+                  Mean_phi_Quotient_(0),
+                  Mean_phi_Fraction_(0),
+                  Mean_eta_Quotient_(0),
+                  Mean_eta_Fraction_(0),
+                  Mean_roz_Quotient_(0),
+                  Mean_roz_Fraction_(0),
+                  Sigma_z_Quotient_(0),
+                  Sigma_z_Fraction_(0),
+                  Sigma_phi_Quotient_(0),
+                  Sigma_phi_Fraction_(0),
+                  Sigma_eta_Quotient_(0),
+                  Sigma_eta_Fraction_(0),
+                  Sigma_roz_Quotient_(0),
+                  Sigma_roz_Fraction_(0),
+                  FirstLayer_(0),
+                  LastLayer_(0),
+                  ShowerLen_(0),
+                  CoreShowerLen_(0),
+                  E_EM_over_E_Quotient_(0),
+                  E_EM_over_E_Fraction_(0),
+                  E_EM_core_over_E_EM_Quotient_(0),
+                  E_EM_core_over_E_EM_Fraction_(0),
+                  E_H_early_over_E_Quotient_(0),
+                  E_H_early_over_E_Fraction_(0)
```

Comment:
Do these ever get set to their calculated values?

Reply:
Implemented setters in commit 342389f